app/models/miq_worker.rb

```diff
@@ -80,46 +80,25 @@ def self.has_minimal_env_option?
   self.check_for_minimal_role = true
   self.required_roles         = []
 
-  def self.server_scope(server_id = nil)
+  def self.server_scope
     return current_scope if current_scope && current_scope.where_values_hash.include?('miq_server_id')
-    if server_id.nil?
-      server = MiqServer.my_server
-      server_id = server.id unless server.nil?
-    end
-    where(:miq_server_id => server_id)
-  end
-
-  CONDITION_CURRENT = {:status => STATUSES_CURRENT}
-  def self.find_current(server_id = nil)
-    server_scope(server_id).where(CONDITION_CURRENT)
+    where(:miq_server_id => MiqServer.my_server&.id)
   end
 
-  def self.find_current_in_region(region)
-    in_region(region).where(CONDITION_CURRENT)
+  def self.find_current
+    server_scope.where(:status => STATUSES_CURRENT)
   end
 
-  def self.find_current_in_my_region
```

Comment:
This is called [here](https://github.com/ManageIQ/manageiq/blob/d46c321f350add809d23c9224ebf4a5280b6e7f5/app/models/vmdb_database_connection.rb#L105)

Reply:
thanks - I circled back with `grep` and didn't see any other nuances